components/text-utilities/text-utilities.html

```diff
@@ -0,0 +1,155 @@
+          </li>
+          <li class="sidebar-label"><label>Components</label></li>
+          <li class="sidebar-li">
+            <a class="sidebar-link" href="../alert/alert.html">Alert</a>
+          </li>
+          <li class="sidebar-li">
+            <a class="sidebar-link" href="../avatar/avatar.html">Avatar</a>
+          </li>
+          <li class="sidebar-li">
+            <a class="sidebar-link" href="../badge/badge.html">Badge</a>
+          </li>
+          <li class="sidebar-li">
+            <a class="sidebar-link" href="../button/button.html">Buttons</a>
+          </li>
+          <li class="sidebar-li">
+            <a class="sidebar-link" href="../card/card.html">Cards</a>
+          </li>
+          <li class="sidebar-li">
+            <a class="sidebar-link" href="../image/image.html">Images</a>
+          </li>
+          <li class="sidebar-li">
+            <a class="sidebar-link" href="../input/input.html">Input</a>
+          </li>
+          <li class="sidebar-li">
+            <a class="sidebar-link" href="../list/list.html">Lists</a>
+          </li>
+          <li class="sidebar-li">
+            <a class="sidebar-link" href="./text-utilities.html">Text Utilities</a>
+          </li>
+        </ul>
+      </div>
+      <!-- sidebar ends here -->
+
+      <main class="main-content-container">
+        <h1>Text Utilities</h1>
+        <P>Use Text Utilities to present your content clearly and efficiently.</P>
+        <p>Falcon UI uses Montserrat font.</p>
+
+        <h2>Headings</h2>
+        <p>You can directly use the elements h1, h2, h3, h4</p>
+        <div class="display-container">
+          <h1>This is a Heading</h1>
+          <h2>This is a Heading</h2>
+          <h3>This is a Heading</h3>
+          <h4>This is a Heading</h4>
+        </div>
+        
+        <pre><code class="display-code language-html">
+                &lt;h1&gt;This is a Heading&lt;/h1&gt;
+                &lt;h2&gt;This is a Heading&lt;/h2&gt;
+                &lt;h3&gt;This is a Heading&lt;/h3&gt;
+                &lt;h4&gt;This is a Heading&lt;/h4&gt;
+        </code></pre>
+
+        <h2>Text</h2>
+        <p>You can use class <code>p-md</code> for font-size of <code>p</code> element, for other sizes, use class <code>p-lg p-md p-sm</code> </p>
+        <div class="display-container">
+            <p class="p-lg">This is a Text</p>
+            <p class="p-md">This is a Text</p>
+            <p>This is a Text</p>
+            <p class="p-sm">This is a Text</p>
+            <small>This is small</small>
+        </div>
+        
+        <pre><code class="display-code language-html">
+            &lt;p class="p-lg"&gt;This is a Text&lt;/p&gt;
+            &lt;p class="p-md"&gt;This is a Text&lt;/p&gt;
+            &lt;p&gt;This is a Text&lt;/p&gt;
+            &lt;p class="p-sm"&gt;This is a Text&lt;/p&gt;
+            &lt;small&gt;This is small&lt;/small&gt;
+        </code></pre>
+
```

Comment:
do use comments to elaoborate your code to the end developer reading you code!!

Reply:
i tried using class names which will give the idea